server/GameOn/GameOn.Tournaments/Controllers/TournamentsResultsController.cs

```diff
@@ -32,16 +34,29 @@ public TournamentsResultsController(ILogger<TournamentsResultsController> log, T
 
             // v2
 
-            // 1. Recalculate Scores for each player in the Result
-            var newScores = await _tournaments.CalculatePlayerScores(tenantId, tournamentId, result);
-
-            // 2. Recalulate Rankings for all players in the Tournament
-            var tournament = await _tournaments.UpdatePlayerRankScores(tenantId, tournamentId, newScores);
-
-            // 3. Message results topic
-            // TODO
-
-            return tournament;
+            try
+            {
+                // 1. Recalculate Scores for each player in the Result
+                var newScores = await _tournaments.CalculatePlayerScores(tenantId, tournamentId, result);
+
+                // 2. Recalulate Rankings for all players in the Tournament
+                var tournament = await _tournaments.UpdatePlayerRankScores(tenantId, tournamentId, newScores);
+
+                // 3. Message results topic
+                // TODO
+                
+                return tournament;
+            }
+            catch (NotFoundException ex)
```

Comment:
Catching exceptions like this allows us to return the correct HTTP status. I will demo this next week.